src/astro/databases/postgres.py

```diff
@@ -0,0 +1,31 @@
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+
+from astro.databases.base import BaseDatabase
+from astro.sql.tables import Table
+
+DEFAULT_CONN_ID = PostgresHook.default_conn_name
+
+
+class PostgresDatabase(BaseDatabase):
+    """
+    Handle interactions with Postgres databases. If this class is successful, we should not have any Postgres-specific
+    logic in other parts of our code-base.
+    """
+
+    def __init__(self, conn_id: str = DEFAULT_CONN_ID):
+        super().__init__(conn_id)
+
+    @property
+    def hook(self):
+        """Retrieve Airflow hook to interface with the Postgres database."""
+        return PostgresHook(postgres_conn_id=self.conn_id)
+
+    def get_table_qualified_name(self, table: Table) -> str:
+        """
+        Return table qualified name. This is Database-specific.
+        For instance, in Sqlite this is the table name. In Snowflake, however, it is the database, schema and table
+
+        :param table: The table we want to retrieve the qualified name for.
+        """
+        name: str = table.name
+        return name
```

Comment:
```suggestion
        return table.name
```

Reply:
I did this because of MyPy, mypy thinks that the function returns the 'Any' if we don't explicitly give type here. Any better way to do this.

Reply:
The following should work too
```suggestion
        return str(table.name)
```